js/core/clusters/clusters.js

```diff
@@ -85,6 +85,7 @@ define([
         }
 
         config.cluster = cluster;
+        window.cluster = cluster;
```

Comment:
If window.cluster is introduced, config.cluster must be removed from code to avoid duplication.